config/300-clusterimagepolicy.yaml

```diff
@@ -125,8 +125,12 @@ spec:
                               properties:
                                 issuer:
                                   type: string
+                                issuerRE:
```

Comment:
Suggestion: perhaps it is just me, but wouldn't it read better to use `issuerRegExp` instead of `issuerRE`?